compiler/rustc_expand/src/proc_macro_server.rs

```diff
@@ -525,6 +525,12 @@ impl server::TokenStream for Rustc<'_, '_> {
             ast::ExprKind::Lit(l) => {
                 Ok(tokenstream::TokenStream::token_alone(token::Literal(l.token_lit), l.span))
             }
+            ast::ExprKind::IncludedBytes(sp, _, bytes) => {
+                let string = bytes.escape_ascii().to_string();
+                let token_lit = token::Lit::new(token::ByteStr, Symbol::intern(&string), None);
+
+                Ok(tokenstream::TokenStream::token_alone(token::TokenKind::Literal(token_lit), *sp))
```

Comment:
Could you add a method converting `IncludedBytes` to its canonical token representation, and then use it both here and in AST pretty printing?